Due to the long-standing difficulty in assembling repetitive DNA, in-depth analyses of centromeric sequences have been limited \cite{Plohl2014}. Based on the high-quality genomes of the three medaka inbred strains which were recently assembled with PacBio long reads, this study conducted in-depth analyses of the centromeric sequences. This is, to the author's knowledge, the first in-depth study of centromeric sequence organization in fish species.

This study revealed the presence of four subfamilies of the centromeric satellite which are possessed by basically different subset of chromosomes. The presence of centromeric satellite subfamilies have been also observed in human, in which, similarly to the medaka subfamilies, each subfamily belongs to the generally different subsets of chromosomes \cite{Alexandrov2001}. It was also observed that the subfamilies had different preference of the centromeric positions in chromosomes. For example, the medaka SF\,2 showed clear preference for acrocentric chromosomes. This is analogous to the human acrocentric chromosomes (chr.\,13, 14, 21, 22) which share highly identical alpha-satellite \cite{Willard1991}. The human acrocentric chromosomes possess nucleolus organizers region (NOR) at their short arms and present in close proximity in the nucleolus, where frequent sequence exchange are believed to occur \cite{Willard1991}. Although the presence of NORs have not been investigated, the same mechanism may work in medaka and produce the high centromeric sequence similarity in the acrocentric chromosomes.

This study also revealed that the centromeric satellites of the same chromosome are largely conserved among the three strains which are estimated to have separated 18 and 25 million years ago \cite{Setiamarga2009}. This conservation is consistent with a previous comparative study of 282 animal and plant species that observed the centromeric satellite are conserved among species which separated within 50 million years ago \cite{Melters2013}. On the other hand, the sequence organization including linear ordering of satellites were not conserved among the strains, suggesting that the centromeric sequences have evolved independently after the separations.

The medaka genome assemblies using PacBio long reads captured centromeric arrays in many chromosomes, including some long arrays over 100 kb. However the amount of centromeric repeats identified in the contigs that were anchored to the chromosomes was substantially below the estimated genomic abundance of the centromeric repeats in all the strains. Indeed large amount of the repeats were found in unanchored contigs. BAC/fosmid-end reads and Hi-C contact frequency data were used for the Hd-rR genome assembly, which successfully anchored a number of contigs containing centromeric repeats and resulted in higher portion of centromeric repeats to be anchored despite shorter read length than HSOK. This result emphasizes the complementary power of other long-range information methods in addition to the long read sequencing. The sum of the centromeric repeats in anchored and unanchored contigs did not reach the estimated genomic abundance, suggesting uncertain amount of the repeats were lost in the assembly process. Highly-homogenized HORs which might exist in the inner centromeric regions may have caused the under-represtation of centromeric repeats in the assembled contigs.

The centromeric sequence organization of three medaka strains revealed in this study provides insights into centromeric sequence evolution. Also, the characterized centromeric sequences provides a important basis for understanding the possible contribution of DNA sequences to the centromere specification and function.
