The centromere is a chromosomal region where kinetochore forms and plays a critical role for proper chromosome segregation in mitosis and meiosis. The centromere is characterized by the presence of centromere-specific histone H3 variant CENH3 (also known as CENP-A). In the majority of species studied so far, the centromere is comprised of repetitive DNA \cite{Plohl2014}. Despite its fundamental biological importance, the mechanism how the position of the centromere is specified are still insufficiently understood \cite{McKinley2016}.

Repeat-based regional centromere is the common structure in eukaryote species. Many species possess satellite DNA specific to the species. A well-known example is 171-bp AT-rich alpha-satellite observed in human and many other primate species. In the human genome, tandemly-repeated alpha-satellite comprise hundred kilobase- to megabase-sized arrays in each chromosome. The alpha-satellite monomer exhibits high divergence up to 40\% within a species. In core centromeric regions, multiple monomers comprise a higher-order repeat units which themselves iterate tandemly with extremely high identity ($>$95\%); this structure is called higher-order repeats (HORs). Another major component of eukaryote centromeres is retrotransposons. Retrotransposon-based centromeres were widely observed in plant species. Satellites and retrotransposons are not mutually exclusive, rather intermingled structure of them are commonly observed. These regional centromeres are flanked by heterochromatic pericentromeres. Centromeres and pericentromeres are characterized by distinct chromatin structures which are regulated by different sets of epigenetic marks (discussed in detail below).

While repeat-based regional centromeres are the most common structure in eukaryotes, some species possess different types of centromeres. Budding yeast \textit{Saccharomyces cerevisiae} has specific $\sim$125-bp sequences at the centromere; this is called a ``point centromere''. In nematode \textit{C. elegans} and some insects and plants, the spindle microtubules attach all along a chromosome and the entire chromosome functions as a centromere, called ``holocentromere''.

Fundamental understanding of centromeric sequence characteristics was established by a number of early studies in 1980's and 1990's, mainly targeting human centromeres, based on experimental methods such as genomic fragmentation by restriction enzymes, pulsed field gel electrophoresis and DNA hybridization. The understandings include approximate estimation of repeat array size and its divergence among individuals \cite{Oakey1990}, the presence of chromosome-specific alpha-satellite HOR patterns \cite{Willard1987} and super-chromosomal subfamilies of alpha-satellites \cite{Alexandrov2001}. It was also revealed that alpha-satellite is widely shared within primate species and even HOR patterns are shared with closely-related species including chimpanzee, gorilla and orangutan \cite{Willard1991}.

Although the basic characteristics of centromeric sequences were revealed by these early studies, understanding detailed sequence organization of the centromere has been challenging in many species, due to the difficulty of assembling its highly-repetitive sequences. In the human genome project, which declared completion in 2003, large portion of centromeric sequences were remained as huge gaps. Whereas divergent monomeric portion around pericentromeres were assembled in many chromosomes, only a limited number of chromosomes reached more homogeneous HOR regions \cite{M.KatharineRuddand2004, She2004}. Nevertheless, subsequent analyses on a few chromosomes that did assemble till the core centromeric regions offered sequence landscapes with never-seen resolution \cite{Schueler2001, Ross2005, Nusbaum2006, Rudd2006}.

Although second-generation sequencers (SGSs) represented by Illumina and 454 accomplished a number of genome assemblies \cite{Schatz2010}, they achieved virtually no improvement in centromeric sequence assembly because of their short read length, emphasizing that centromere studies need specific approaches considering its sequence characteristics. Nonetheless, their high throughput sequencing combined with chromatin immunoprecipitation (ChIP-seq) facilitated identification of centromere-associated sequences in many species \cite{} and characterization of functional regions in the assembled centromeric sequences \cite{Hayden2013}.

A number of computational studies were conducted using Sanger and Illumina whole genome shotgun (WGS) sequencing data, some of which made remarkable achievement. Some studies identified candidate centromeric satellite sequences from WGS data \cite{Alkan2011, Melters2013}, whereas others identified novel HOR patterns from assembled sequences \cite{Rosandic2003} or from WGS data \cite{Alkan2007}.
Melters \textit{et al}. \cite{Melters2013} identified candidate centromeric satellite sequences of 282 species (204 animal and 78 plant species) using WGS data from various sequencing platforms, mainly from Sanger and Illumina. They based on the assumption that the most abundant tandem repeat in a genome derives from centromeric sequences, which is true for most species whose centromeric sequences has been previously characterized. This study revealed that centromeric satellites from various eukaryotic species do not share common properties such as repeat unit length, GC content or genomic abundance and that centromeric satellite sequences are conserved among only closely-related species of within 50 million years after separation. These results confirmed a traditional view that centromeric sequences evolves rapidly, independently of the rest of genomic sequences \cite{Henikoff2001}.
Another remarkable computational study is from Miga \textit{et al}. \cite{Miga2014}, in which they generated centromeric array sequences of each human chromosome, using graph-based probabilistic models constructed from Sanger WGS reads. Although the generated sequences do not guarantee large-scale ordering of the satellite sequences, they adequately represent local ordering, thus provides useful scaffolds for downstream analysis like sequencing read mapping. The generated centromeric sequences, moreover, has been included in the latest human reference genome (GRCh38).
