\subsection*{Sequencing and genome assembly}
Sequencing and assembling the three medaka strain genomes were carried out by Kazuki Ichikawa and Jun Yoshimura in the same laboratory. The detail of the methods will be described in Ichikawa \textit{et al}. (unpublished). Here a brief overview of the methods is given.

The genomes were sequenced with PacBio SMRT sequencing and were assembled into contigs using FALCON assembler. The contigs were then mapped to the chromosomes using genetic markers. As to the Hd-rR genome, the contigs were scaffolded using BAC- and fosmid-end pair sequencing reads and a number of unanchored contigs were positioned into the chromosomes using Hi-C contact frequency data. The whole process produced 734 Mb, 678 Mb and 744 Mb assembly of Hd-rR, HNI and HSOK genomes, respectively.

\subsection*{Validating centromeric sequence assembly}


\subsection*{Estimating genomic abundance of centromeric repeats}
In order to minimize the effect of high error rate of PacBio sequencing on abundance estimation of the centromeric repeats, only high quality subreads were used for this step. Specifically, subreads were filtered with the criteria that average base quality over the all bases $>$10. Also, subreads shorter than 1 kb were excluded. The filtered subreads were then scanned by RepeatMasker with a sensitive setting using the medaka representative centromeric satellite monomer sequence as a custom library. Genomic fraction of the medaka centromeric satellite for each strain was estimated by the ratio of total amount of masked centromeric satellite in the total length of the filtered subreads (Table \ref{centromeric_repeat_genomic_abundance}).

\subsection*{Revealing centromeric repeat distribution and centromeric positions}
The three medaka strain genomes were searched for the medaka centromeric satellite by RepeatMasker with sensitive setting. For those chromosomes that have $>$1 kb centromeric repeat, positions of the centromeres were classified employing the nomenclature defined in Levan \textit{et al}. (1964). The nomenclature divides a chromosome equally into eight portions and classify the chromosome by the position of the centromere from the two most inners to the two most outers as metacentric, submetacentric, subtelocentric and acrocentric. In this study, chromosomes were classified into a portion that contains the largest amount of centromeric repeats.

\subsection*{Inter-chromosomal centromeric sequence comparison}
Centromeric repeat arrays in each chromosome of the three strains were decomposed into satellite monomers by RepeatMasker with sensitive setting. The monomer sequences within each chromosome were then clustered into groups of $>$85\% sequence similarity by DNACLUST \cite{}. For those clusters that have $\geq$10 members, the monomer with the longest sequence in the cluster was chosen as the representative monomer of the cluster. All-vs-all pairwise alignment of the chromosome-representative monomers along with the representative monomer identified by Melters \textit{et al}. was performed by EMBOSS needle program. The distance between a pair of two monomers was calculated as below:

\[
  \mbox{distance} = 1 - \frac{\mbox{number of matched bases}}{\mbox{length of shorter monomer}}
\]

Based on this distance, hierarchical clustering of the chromosome-representative monomers were performed by "hclust" function in R with "ward.D2" method.

% TODO: Minimize the problem of uncertainty on monomer interval determination
